src/Umbraco.Web.UI.Client/src/views/packager/views/repo.controller.js

```diff
@@ -218,21 +185,15 @@
                     },
                     error)
                 .then(function(result) {
-
-                        if (result.postInstallationPath) {
```

Comment:
Same deal as before, removing this logic now means that correct package installation cannot occur